docs/src/components/docs/SideNav.js

```diff
@@ -78,7 +78,7 @@ const UtilitiesLinks = () => {
   const nodes = sortByTitle(utilities.nodes)
 
   return nodes.map(({ frontmatter: { title }, fields: { slug } }) => (
-    <ComponentLink key={title} href={`/${slug}`}>
```

Comment:
@with-heart if you use this format without the trailing backslash, you should carry it over to the rest of the website with https://www.gatsbyjs.org/packages/gatsby-plugin-remove-trailing-slashes/. This is for SEO purposes to prevent 301 redirects, it needs to be consistent 👍